### PR TITLE
Add wagtailuserbar to the base.html

### PR DESCRIPTION
**Pull request by kurtw**
- Makes it so when viewing a page while logged in will give you page controls

It's a convenience but a nice one at that.

@richaagarwal 
@kave 
@jimmynotjim 
@anselmbradford

### CODE REVIEW

**Comment by richaagarwal** [comment]
:+1:

File: cfgov/jinja2/v1/_layouts/base.html [hunk 1]
```diff
@@ -159,6 +159,9 @@
     {% include 'templates/header.html' %}
 {% endblock header %}
 
+<!-- WAGTAIL USERBAR IN PAGE CONTROLS -->
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_layouts/base.html]: Consider converting to jinja comment as `{# WAGTAIL USERBAR IN PAGE CONTROLS #}` unless there's a reason to show this in the output HTML.

**Comment by jimmynotjim** [comment]
:+1: Looks good

**Comment by kave** [comment]
:+1:

**Pull request merged**